src/plugins/googledrive/server/routes/content-api.js

```diff
@@ -0,0 +1,27 @@
+const routes = require('.');
+
+module.exports = {
+  type: 'content-api',
+  routes: [
+    {
+      method: 'GET',
+      path: '/',
+      handler: 'welcomeController.index',
+    },
+    {
+      method: 'GET',
+      path: '/get',
+      handler: 'operationController.get',
+    },
+    {
+      method: 'POST',
+      path: '/upload',
+      handler: 'operationController.upload',
+    },
+    {
+      method: 'DELETE',
+      path: '/delete',
```

Comment:
I think we need `/delete/:id` for https://github.com/dev-launchers/strapiv4/pull/250#discussion_r1884797691.